metricbeat/module/docker/image/_meta/fields.yml

```diff
@@ -0,0 +1,9 @@
+- name: image
+  type: group
+  description: >
+    image
+  fields:
+    - name: example
+      type: keyword
+      description: >
+        Example field
```

Comment:
This must be completed with the fields, otherwise the system test will not pass.